Graphics/GraphicsEngine/include/PipelineStateBase.hpp

```diff
@@ -388,9 +408,11 @@ class PipelineStateBase : public DeviceObjectBase<BaseInterface, RenderDeviceImp
     RefCntAutoPtr<IShader> m_pDS; ///< Strong reference to the domain shader
     RefCntAutoPtr<IShader> m_pHS; ///< Strong reference to the hull shader
     RefCntAutoPtr<IShader> m_pCS; ///< Strong reference to the compute shader
+    RefCntAutoPtr<IShader> m_pAS; ///< Strong reference to the amplification shader
+    RefCntAutoPtr<IShader> m_pMS; ///< Strong reference to the mesh shader
 
-    IShader* m_ppShaders[5]             = {}; ///< Array of pointers to the shaders used by this PSO
-    size_t   m_ShaderResourceLayoutHash = 0;  ///< Hash computed from the shader resource layout
+    IShader* m_ppShaders[MAX_SHADERS_IN_PIPELINE] = {}; ///< Array of pointers to the shaders used by this PSO
```

Comment:
Can't find where this is defined.

Reply:
in [Graphics\GraphicsEngine\interface\Constants.h](https://github.com/DiligentGraphics/DiligentCore/blob/master/Graphics/GraphicsEngine/interface/Constants.h#L54)